doc/development/02devcontainer.md

```diff
@@ -0,0 +1,30 @@
+# Devcontainer
+
+We are also supporting a containerized setup(docker/podman). For more information checkout the README inside [.devcontainer](https://github.com/ElektraInitiative/PermaplanT/blob/master/.devcontainer/README.md).
+
+## Background
+
+(Devcontainer)[https://code.visualstudio.com/docs/devcontainers/containers] allows you to
```

Comment:
Markdown is the other way round. Please try to write one sentence per line and keep sentences shorter.